Modix.Services/AutoRemoveMessage/AutoRemoveMessageService.cs

```diff
@@ -41,23 +42,27 @@ public AutoRemoveMessageService(INotificationDispatchService notificationDispatc
         }
 
         /// <inheritdoc />
-        public async Task<EmbedBuilder> RegisterRemovableMessageAsync(IMessage message, IUser user, EmbedBuilder embed)
+        public async Task RegisterRemovableMessageAsync(IUser user, EmbedBuilder embed, Func<EmbedBuilder, Task<IUserMessage>> callback)
         {
-            await NotificationDispatchService.PublishScopedAsync(new RemovableMessageSent()
-            {
-                Message = message,
-                User = user,
-            });
+            if (callback == null)
+                return;
```

Comment:
It feels like this should be an `ArgumentNullException` rather than silently returning.

Reply:
Yup.